doc/SuppTechInfo/gen-coupling.tex

```diff
@@ -0,0 +1,43 @@
+
+Connecting models such that the dynamics at the interface can be described properly by an Interface Model does impose a few conditions on the configuration. For more advanced functionality such as XT3D, it is required that the relative position of the model grids is specified. They should also share the cell faces over which they are connected, such that the resulting grid for the interface can be specified as a valid DISU discretization. Another consequence is that a certain compatibility is required with respect to the active processes and configuration in the individual models: enabling advection (ADV) or dispersion (DSP), for example, only in one of two connected models introduces ambiguity in how to construct the Interface Model for the transport process. Similar arguments hold when two groundwater models are connected and only one of them is configured with a buoyancy package (BUY). Such model setups are currently not supported.
+
+\begin{figure}[!t]
+	\begin{center}
+	\includegraphics[width=0.5\textwidth]{./Figures/InterfaceModel/mt3dms-p10-modelgrid.png}
+	\caption[A plan view of the model grid from MT3DMS problem 10]{A plan view of the model grid for the simulation as defined in MT3DMS problem 10. The red dashed rectangle shows the location of the interface that is used for decomposing this case into a system of coupled GWF and GWT models.}
+	\label{fig:gwtgwt-fullgrid}
+	\end{center}
+\end{figure}
+
+To illustrate the concept of this new coupling, a well known, single-model case study (problem 10 in the MT3DMS manual \cite{zheng1999mt3dms}) is implemented as a coupled system where the connection between the GWT models is handled by the Interface Model framework. This study is part of the official set of examples that comes with every \mf software release. The coefficients for the flow coupling in this case are calculated with the basic GWF Model Exchange module and the following discussion will focus on the GWT-GWT exchange. Figure~\ref{fig:gwtgwt-fullgrid} shows a plan view of the grid and the location of the interface between the submodels. Note that changing only the composition of the grid and leaving the model configuration the same, does not affect the outcome of the simulation\footnote{At the highest level of detail, the outcome does change because the numbering of the grid nodes is different in both scenarios leading to an equivalent but reordered matrix system to be solved. As expexted, these differences turn out to be well within the configured (IMS) solver tolerance.} and therefore makes for a very suitable test case of this coupling.
```

Comment:
"expexted" is not expected

Reply:
Unrelated, but adding a new line after every sentence makes it much easier to review LaTeX as well as Markdown without changing the actual output.